addons/fortify/Cfg3DEN.hpp

```diff
@@ -0,0 +1,27 @@
+class Cfg3DEN
+{
+    class Attributes
+    {
+        class Title;
+        class Combo: Title {
+            class Controls {
+                class Title;
+                class Value;
+            };
+        };
+        class GVAR(presetSelection): Combo {
+            class Controls: Controls {
+                class Title: Title {};
+                class Value: Value {
+                    delete Items;
+                    class ItemsConfig {
+                        path[] = { QGVAR(Presets) };
```

Comment:
No spaces around braces.
```suggestion
                        path[] = {QGVAR(Presets)};
```